README.md

```diff
@@ -32,6 +37,51 @@ Meeting Agenda templates
 - Meeting - Sprint Retrospective, Review, and Planning
 - Meeting - Sprint Open Topic Session 
 
+## Running the project locally
+To start the project locally type
+
+
+The following list of things are required to use the software:
+
+- npm
+  ```sh
+  npm install npm@latest -g
+
+- Clone the repo
+   ```sh
+   git clone https://github.com/chingu-voyages/v46-tier3-team-33/tree/readme4
+   ```
+- Install NPM packages
+   ```sh
+   npm install
+   ```
+- Create .env at the root of repo
+- Enter database url in .env  mongodb+srv://varchodi:greenov@cluster0.elkzdh3.mo
+
+
+Now you are ready to run the project locally.  In a terminal type
+
+- type
+ ``` sh
+ cd frontend```  
+ then 
+ 
+ ```npm install```
+  once you are in the front end directory
+
+```npm run dev
+```
+- this will run the frontend
```

Comment:
Could add that the URL that is shown is what they need to open in a browser. localhost etc